### PR TITLE
chore: rename `streaming/group_top_n/main1.slt` to `main.slt`

### PR DESCRIPTION
**Pull request by stdrc**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

Previously I named it `main1.slt` because of a limitation of `sqllogictest-rs`, now it's fixed (https://github.com/risinglightdb/sqllogictest-rs/pull/194, https://github.com/risinglightdb/sqllogictest-rs/pull/198).

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

**Review by xxchan — APPROVED** [review]

**Comment by xxchan** [comment]
QueryError: Catalog error: database with name main_slt exists"

**Comment by stdrc** [comment]
Seems the sqllogictest version is not bumped? @xxchan

**Comment by xxchan** [comment]
https://github.com/risingwavelabs/risingwave/blob/23d2f81693bca530613dfa45cdef8c15eba92ab9/ci/Dockerfile#L45

**Comment by xxchan** [comment]
Oh, maybe we need to upgrade for sim.

**Comment by stdrc** [comment]
```
thread '<unnamed>' panicked at /risingwave/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sqllogictest-0.17.0/src/runner.rs:1035:18:
create db failed: Error { kind: Db, cause: Some(DbError { severity: "ERROR", parsed_severity: None, code: SqlState(EXX000), message: "QueryError: Catalog error: database with name main_slt exists", detail: None, hint: None, position: None, where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: None, line: None, routine: None }) }
stack backtrace:
   0: rust_begin_unwind
             at /rustc/249624b5043013d18c00f0401ca431c1a6baa8cd/library/std/src/panicking.rs:597:5
   1: core::panicking::panic_fmt
             at /rustc/249624b5043013d18c00f0401ca431c1a6baa8cd/library/core/src/panicking.rs:72:14
   2: core::result::unwrap_failed
             at /rustc/249624b5043013d18c00f0401ca431c1a6baa8cd/library/core/src/result.rs:1653:5
   3: core::result::Result<T,E>::expect
             at /rustc/249624b5043013d18c00f0401ca431c1a6baa8cd/library/core/src/result.rs:1034:23
   4: sqllogictest::runner::Runner<D,M>::run_parallel_async::{{closure}}
             at ./.cargo/registry/src/index.crates.io-6f17d22bba15001f/sqllogictest-0.17.0/src/runner.rs:1032:13
   5: risingwave_simulation::slt::run_parallel_slt_task::{{closure}}
             at ./src/tests/simulation/src/slt.rs:237:10
   6: risingwave_simulation::main::{{closure}}::{{closure}}::{{closure}}
             at ./src/tests/simulation/src/main.rs:246:51

```

It's using 0.17, weird.

**Comment by stdrc** [comment]
> Oh, maybe we need to upgrade for sim. It uses lib instead of bin.

Submitted a new PR for sqllogictest-rs: https://github.com/risinglightdb/sqllogictest-rs/pull/198

**Review by xxchan — APPROVED** [review]

File: ci/Dockerfile
Comment (xxchan) [inline ci/Dockerfile:1]: need to update BUILD_ENV_VERSION

Comment (xxchan) [reply ci/Dockerfile:1]: Well, maybe we can add a comment in this dockerfile about that..

Comment (stdrc) [reply ci/Dockerfile:1]: Fixed.

**Review by BugenZhao — APPROVED** [review]

**Comment by BugenZhao** [comment]
Why `audit` keeps failing? 👀

**Pull request merged**